client/controllers/incidentReports/annotatedContent.coffee

```diff
@@ -27,7 +30,11 @@ Template.annotatedContent.onDestroyed ->
 Template.annotatedContent.helpers
   annotatedContent: ->
     selectedAnnotationId = Template.instance().data.selectedAnnotationId.get()
-    annotateContentWithIncidents(@content, @incidents.fetch(), selectedAnnotationId)
+    annotateContentWithIncidents(@content, @incidents.fetch().map((incident)->
+      if not incident.accepted
+        incident.uncertainCountType = true
```

Comment:
Is `uncertainCountType` necessary? It seems like we could just be checking whether an incident is accepted everywhere we are using it instead?  If we did go this route we would need to refactor in a few other places.